<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/settings.ico</normaloff>:/icon/settings.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="path_lb_server">
       <property name="text">
        <string>Default path </string>
       </property>
       <property name="buddy">
        <cstring>path_le_server</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="path_le_server"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pick_pb_server">
       <property name="text">
        <string>Pick</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="port_lb_server">
       <property name="text">
        <string>Default Port </string>
       </property>
       <property name="buddy">
        <cstring>port_le_server</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="port_le_server"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="language_lb">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="language_cb">
       <item>
        <property name="text">
         <string notr="true">English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">汉语</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="cache_lb">
       <property name="text">
        <string>Cache size (need restart)</string>
       </property>
       <property name="buddy">
        <cstring>cache_cb</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cache_cb">
       <item>
        <property name="text">
         <string>4096 byte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8192 byte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12288 byte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16384 byte</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Need confirm?</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QRadioButton" name="yes_rb">
        <property name="text">
         <string>Yes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="no_rb">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="ok_pb">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="apply_pb">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="cancle_pb">
     <property name="text">
      <string>Cancle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>path_le_server</tabstop>
  <tabstop>pick_pb_server</tabstop>
  <tabstop>port_le_server</tabstop>
  <tabstop>cache_cb</tabstop>
  <tabstop>yes_rb</tabstop>
  <tabstop>ok_pb</tabstop>
  <tabstop>apply_pb</tabstop>
  <tabstop>cancle_pb</tabstop>
  <tabstop>no_rb</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancle_pb</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
